integration-tests/contract-originate-contract-with-sapling.spec.ts

```diff
@@ -1,29 +1,61 @@
 import { CONFIGS } from './config';
+import { Protocols } from '@taquito/taquito';
 import { SaplingStateAbstraction } from 'taquito/src/contract/sapling-state-abstraction';
 import {
+  saplingContractDouble,
+  saplingContractDrop,
+  saplingContractSend,
   saplingContractStateAsArg,
   saplingContractPushSaplingState,
+  saplingContractUseExistingState,
   saplingContractDoubleJProto,
   saplingContractDropJProto,
   saplingContractSendJProto,
   saplingContractUseExistingStateJProto,
 } from './data/sapling_test_contracts';
 
-CONFIGS().forEach(({ lib, rpc, setup }) => {
+CONFIGS().forEach(({ lib, rpc, protocol, setup }) => {
   const Tezos = lib;
+  const jakartanetAndMondaynet = protocol === Protocols.ProtoALpha || protocol === Protocols.PtJakart2 ? test: test.skip;
+  const ithacanet = protocol === Protocols.Psithaca2 ? test : test.skip;
 
   interface StorageType {
     left: SaplingStateAbstraction;
     right: SaplingStateAbstraction;
   }
 
-  describe(`Test origination of contracts with sapling using: ${rpc}`, () => {
+  describe(`Test contract origination with sapling through contract api using: ${rpc}`, () => {
     beforeEach(async (done) => {
       await setup();
       done();
     });
 
-    test('Originates a Sapling Double contract', async (done) => {
+    ithacanet('Originates a Sapling Double contract', async (done) => {
```

Comment:
ithacanet should be removed

Reply:
ithacanet removed and Jakarta/mondaynet set to just test.